.github/workflows/protocol-devchain.yml

```diff
@@ -0,0 +1,54 @@
+name: 'CoreContracts: Devchain For Contract Release'
+run-name: 'CoreContracts: Devchain For Contract Release'
+
+on:
+  schedule:
+    # monthly on 1 at 0:00 UTC
+    - cron: 0 0 1 * *
+  workflow_dispatch:
+
+jobs:
+  generate-protocol-devchain:
+    name: Generate protocol devchain used in celo-monorepo.yml workflow
+    runs-on: ['self-hosted', 'monorepo-node18']
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - tag: core-contracts.v9
```

Comment:
Not sure the node12 compatibility is still needed.

Reply:
we can remove it later